### PR TITLE
maintenance

### LINKED ISSUE #100: button to get links to sections
image
like the ones on quiltmc.org!
@simpledemonn I hear you like making these

### PR DESCRIPTION
**Pull request by ix0rai**
- fix typos
- remove :P
- fix #100 

---
See preview on Cloudflare Pages: https://preview-103.developer-wiki.pages.dev

### CODE REVIEW

File: src/lib/translations/en/application.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-	"title": "Quilt Developer Wiki :P",
+	"title": "Quilt Developer Wiki",
```
Comment (Pyrofab) [inline src/lib/translations/en/application.json:2]: Hot take : I think the silly is fine to keep as long as we don't have real content for this page \:P

Comment (ix0rai) [reply src/lib/translations/en/application.json:2]: imo the dev wiki seems to get a lot of viewership, and it's better to have it looking spiffy

Comment (Pyrofab) [reply src/lib/translations/en/application.json:2]: Then... could we maybe get a small welcome thingy just to make the page seem not empty ?

Comment (ix0rai) [reply src/lib/translations/en/application.json:2]: yes

**Comment by c-leri** [comment]
Just adding a comment to signal that
- The typo is corrected and the silly is removed in #106
- The welcome thingy is added in #104
- Issue #100 is fixed in #108 without adding fontawesome as a dependencies

Merging these would thus make this one obsolete

**Comment by ix0rai** [comment]
got it, thanks!

**Pull request closed without merging**